src/Core/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## NOT RELEASED
 
+## 1.18.0
```

Comment:
thanks for updating the changelog, but this is not how it works.

- you should not add the version (just add a `### Added` section under `## NOT RELEASED`)
- You should update the `CHANGELOG.md` inside `src/Service/CognitoIdentityProvider/` folder
- try to be consistent with other changes (You'll find examples is the file above) like 
```
Added operation `AdminAddUserToGroup`
Added operation `AdminRemoveUserFromGroup`
...
```


thanks for you help

Reply:
Thanks for the explanation. Updated